core/src/main/java/nl/nn/adapterframework/jdbc/JdbcFacade.java

```diff
@@ -212,12 +212,18 @@ public void setJmsRealm(String jmsRealmName) {
 
 	/**
 	 * Returns the name and location of the database that this objects operates on.
-	 * 
+	 *
 	 * @see nl.nn.adapterframework.core.HasPhysicalDestination#getPhysicalDestinationName()
 	 */
 	@Override
 	public String getPhysicalDestinationName() {
 		try {
+			DataSource dataSource;
+			try {
+				dataSource = getDatasource();
+			} catch (Exception e) {
+				return "no datasource found for datasourceName ["+getDatasourceName()+"]";
+			}
```

Comment:
wanneer dataSource null is komt hij niet door `if(getDatasource() instanceof TransactionalDbmsSupportAwareDataSourceProxy) {` Ik denk dat deze check meer de getConnection thuis hoort, of anders voor het ophalen van de connectie in deze methode.